### PR TITLE
Fixup category assignments in the Galnet monitor and create localizable category filters

### LINKED ISSUE #599: Galnet (for Beyond) - proposal
The category assignments for Galnet news haven't been working since fdev revamped the galnet feed.
I didn't see the Galnet code listed as being scheduled for an update, so... 

I have revised the code to sort the galnet news into 4 categories;
1) Article
2) Starport Status Update
3) Week in Review 
4) Community Goal
Article being general galnet news, the other categories are self explanatory.

I have mocked up a PR branch; https://github.com/rodan123/EDDI/tree/feature/galnet30

The category assignment is all in _GalnetMonitor/GalnetMonitor.cs_ 

I also did a small optional change that adds a `ReplaceStrings (string)` function to _GalnetMonitor/GalnetFeedItemNormalizer.cs_ and it currently replaces the word **Thargoid** with **Thargoyd** to attempt a better pronunciation than the current Thar-go-id. I noted below that in a commented section how I am using more substitutions to get better pronunciations of dates in the news.

So far It has been working pretty well for me and I offer it here for consideration to merge with the codebase.

### PR DESCRIPTION
**Pull request by Tkael**
Fix #599.

### CODE REVIEW

File: GalnetMonitor/GalnetMonitor.cs [hunk 1]
```diff
@@ -230,126 +229,41 @@ public void HandleProfile(JObject profile)
...
-                {
-                    return "Starport Status Update";
-                }
+                return "Community Goal";
+            }
 
-                return "Article";
+            if (title.StartsWith(GetGalnetResource("galnetCategoryStarportStatus")))
+            {
+                return "Starport Status Update";
```
Comment (richardbuckle) [inline GalnetMonitor/GalnetMonitor.cs]: Needs localization

File: GalnetMonitor/GalnetMonitor.cs [hunk 2]
```diff
@@ -230,126 +229,41 @@ public void HandleProfile(JObject profile)
...
             }
-            else
+
+            if (title.StartsWith(GetGalnetResource("galnetCategoryWeekInReview")))
             {
-                return "Article";
+                return "Week in Review";
```
Comment (richardbuckle) [inline GalnetMonitor/GalnetMonitor.cs]: Needs localization

**Comment by Tkael** [comment]
On hold for review & rebasing.

**Comment by richardbuckle** [comment]
Let's get 3.0.2 out and address this in 3.1.

**Comment by Tkael** [comment]
From prior to EDDN fixes. Needs rebasing prior to testing.

**Comment by richardbuckle** [comment]
Om reflection I may be wrong that the strings returned by `GalnetMonitor.assignCategory()` need localisation. @Tkael can you confirm please?

If they are in fact just used as internal symbols then maybe an enum would be a better fit.

**Comment by Tkael** [comment]
They are returned to the user, so I'm adding them to the resx.

**Comment by Tkael** [comment]
Closing to resubmit rebased version.

**Pull request closed without merging**